CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+Welcome. We're really glad you're here. [MuseScore](https://musescore.org/) is exceptional software (yes, we're biased) created, updated and used by many.
+
+It has thorough, organized documentation that contains all the resources you�ll need to help keep MuseScore great. A few of these resources are linked at the bottom. This is a shortened and condensed version to help developers new to MuseScore get started. 
+
+## Getting Started - The Basics
+
+### See MuseScore in action
+* [Install](https://musescore.org/en/handbook/3/installation) it
```

Comment:
if you drop the `3` here (plus one of the adjacent `/`) the URL will redirect to whatever version is current at the time. As this PR is for master, thus would be 4 by the time it gets released.
Same for URLs to the handbook furter down

Reply:
all 3s and 'en's dropped from links